local-volume/provisioner/pkg/common/common.go

```diff
@@ -122,6 +126,25 @@ func CreateLocalPVSpec(config *LocalPVConfig) *v1.PersistentVolume {
 	}
 }
 
+// GetVolumeConfigFromMountedConfigMap gets volume configuration from the mounted configmap volume,
+func GetVolumeConfigFromMountedConfigMap() (map[string]MountConfig, error) {
+	volumes := make(map[string]string)
+	files, err := ioutil.ReadDir(ProvisionerConfigPath)
+	if err != nil {
+		return nil, err
+	}
+	for _, file := range files {
+		if !file.IsDir() {
+			b, err := ioutil.ReadFile(path.Join(ProvisionerConfigPath, file.Name()))
```

Comment:
A better variable name could be "fileContents"

Reply:
done